docker/docker_APP/run.txt

```diff
@@ -0,0 +1,10 @@
+# RUN CONTAINER
```

Comment:
It's better to put such instructions into Readme.md with appropriate [markdown](https://guides.github.com/features/mastering-markdown/) so that github will automatically recognize it and show in UI.